### PR TITLE
[BottomSheet] Add API to flashScrollIndicators

### PR DESCRIPTION
**Pull request by codeman7**
## The problem
Currently `MDCBottomSheet` does not indicate to users that there is scrollable content by default. If a client is using this component and want to inform their users that the content is scrollable they currently have no way to do that.

## The fix
Add a behavioral flag for when clients want to `flashScrollIndicators` so that clients can opt into this behavior if they need it. This is recommended by Apple when:

> [You should call this method whenever you bring the scroll view to front.](https://developer.apple.com/documentation/uikit/uiscrollview/1619435-flashscrollindicators?language=objc).

## Alternatives considered
Calling `flashScrollIndicators` by default.

## Videos
| Before | After | 
| - | - |
|before|after|

### CODE REVIEW

**Comment by yarneo** [comment]
Please provide a description explaining the work for this PR.

File: components/BottomSheet/src/MDCBottomSheetController.h [hunk 1]
```diff
@@ -45,6 +45,14 @@
  */
 @property(nonatomic, weak, nullable) UIScrollView *trackingScrollView;
 
+/**
+ This property determines if @c showFlashIndicators is called by default when @c
+ MDCBottomSheetController calls @c viewDidAppear.
+
+ @note Defaults to @c NO.
+ */
+@property(nonatomic, assign) BOOL showScrollIndicatorsByDefault;
```
Comment (jverkoey) [inline components/BottomSheet/src/MDCBottomSheetController.h]: Consider: `shouldFlashScrollIndicatorsOnAppearance`

File: components/BottomSheet/src/MDCBottomSheetController.m [hunk 2]
```diff
@@ -73,6 +73,14 @@ - (void)viewWillAppear:(BOOL)animated {
   [self.contentViewController.view layoutIfNeeded];
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+  [super viewDidAppear:animated];
+
+  if (self.showScrollIndicatorsByDefault && self.trackingScrollView) {
```
Comment (jverkoey) [inline components/BottomSheet/src/MDCBottomSheetController.m]: The `self.trackingScrollView` check here isn't necessary due to nil handling; please remove.

**Review by jverkoey — APPROVED** [review]
> So if a client is using this component and want to inform their users that the content is scrollable.

I think this part of the PR description may need to be updated.

File: components/BottomSheet/src/MDCBottomSheetController.m (hunk 2 above)
Comment (codeman7) [reply components/BottomSheet/src/MDCBottomSheetController.m]: Done

File: components/BottomSheet/src/MDCBottomSheetController.h (hunk 1 above)
Comment (codeman7) [reply components/BottomSheet/src/MDCBottomSheetController.h]: Done

**Review by jverkoey — APPROVED** [review]

**Pull request merged**